modules/database/sqlite/wp-includes/sqlite/db.php

```diff
@@ -0,0 +1,258 @@
+<?php
+/**
+ * Main integration file.
+ *
+ * @package performance-lab
+ * @since n.e.x.t
+ */
+
+// Require the constants file.
+require_once dirname( dirname( __DIR__ ) ) . '/constants.php';
+
+// Bail early if DATABASE_TYPE is not defined as sqlite.
+if ( ! defined( 'DATABASE_TYPE' ) || 'sqlite' !== DATABASE_TYPE ) {
+	return;
+}
```

Comment:
Related to my last comment, that fatal error led me to discover another problem related to this early return: While the condition here itself is good, we need to really make sure none of the database code is loaded if the database type isn't set to `sqlite`.

This is currently the case for all the classes since they are loaded via `require_once`; however the two functions `perflab_sqlite_make_db_sqlite()` and the replacement for `wp_install()` are still loaded. Early returns do not prevent global functions in the same file from becoming available. This is part of what caused the fatal error, since despite this early return the `wp_install()` function below was still callable and would be executed.

Can you please move the functions out of this file, e.g. into a `wp-includes/sqlite/install-functions.php`? That file you can then `require_once` with the other class files, and the problem should be fixed.

Reply:
Done